cloudinit/distros/tests/test_networking.py

```diff
@@ -97,6 +100,33 @@ def test_exists_argument(self, m_udevadm_settle):
         ] == m_udevadm_settle.call_args_list
 
 
+class TestGenerateFallbackConfig:
+    pass
+
+class TestBSDGenerateFallbackConfig:
+    @pytest.fixture
+    def default_netcfg(self):
```

Comment:
from @OddBloke, via IRC: given that `default_netcfg` just returns a static dict, I wonder if it could be a global variable instead?